src/test/java/com/iexec/blockchain/version/VersionServiceTests.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.blockchain.version;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.info.BuildProperties;
+
+class VersionServiceTests {
+
+    @Mock
+    private BuildProperties buildProperties;
+
+    @InjectMocks
+    private VersionService versionService;
+
+    @BeforeEach
+    public void preflight() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings={"x.y.z", "x.y.z-rc"})
+    void testNonSnapshotVersion(String version) {
+        Mockito.when(buildProperties.getVersion()).thenReturn(version);
+        Assertions.assertEquals(version, versionService.getVersion());
+        Assertions.assertFalse(versionService.isSnapshot());
```

Comment:
You can remove `isSnapshot` method instead

Reply:
If we remove `isSnapshot`, why not simply removing `VersionService` and directly use `BuildProperties` when necessary ?
Do we want to go in this direction ?